nsqadmin/static/js/views/topic.hbs

```diff
@@ -81,7 +81,7 @@
             {{#if ../graph_active}}
                 <td class="bold rate" target="{{rate "topic" node topic_name ""}}"></td>
             {{/if}}
-            <td>{{commafy ../channels.length}}</td>
+            <td>{{commafy this/channels.length}}</td>
```

Comment:
this is greek to me :)
I wonder if "this/" is not needed and just "channels.length" would suffice?

Reply:
I'm not sure... I confused myself by working in the wrong part of the template for a while, but there is a global `channels` var so this is at least more explicit.

Reply:
fair enough :) no objection